lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart

```diff
@@ -0,0 +1,55 @@
+part of 'format_comment_rule.dart';
+
+const commentsOperator = ['//', '///', '/*'];
+
+class _Visitor extends RecursiveAstVisitor<void> {
+  final _comments = <CommentInfo>[];
+
+  Iterable<CommentInfo> get comments => _comments;
+
+  void visitComments(AstNode node) {
```

Comment:
Why does the method have no `@override` annotation?

Reply:
It is not an override method.